seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -1174,11 +1174,11 @@
       "id": "bad87fee1348bd9aedf08827",
       "title": "Create a Bulleted Unordered List",
       "description": [
-        "HTML has a special element for creating <code>unordered lists</code>, or bullet point-style lists.",
-        "Unordered lists start with a <code>&#60;ul&#62;</code> element. Then they contain some number of <code>&#60;li&#62;</code> elements.",
+        "HTML has a special element for creating <code>unordered lists</code>, or bullet point style lists.",
+        "Unordered lists starts with an opening <code>&#60;ul&#62;</code> element, followed by any number of <code>&#60;li&#62;</code> elements. Finally, end the unordered list with a closing <code>&#60;/ul&#62;</code>",
```

Comment:
I think "lists starts" is a typo.

"Finally, end the unordered list with..." is imperative.  I think what you mean to say is "Finally, unordered lists close with a `</ul>`"